VideoLocker/src/androidTest/java/org/edx/mobile/test/ConfigTests.java

```diff
@@ -219,6 +223,33 @@ public void testGoogleConfig() {
         assertTrue(config.getGoogleConfig().isEnabled());
     }
 
+
+    public void testNotificationNoConfig() {
```

Comment:
Let's call this the "parse" config, not the notifications config, since we may end up using another notification backend. In general, let's isolate parse as much as possible.